.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -1,16 +1,17 @@
 <!--
 
-Make sure to run `msbuild /t:pack /v:m` in the repository root when
-you have any of the following changes that affect auto-generated files. Otherwise, the CI build will fail.
+Make sure you have read the contribution guidelines: 
+- https://docs.microsoft.com/contribute/dotnet/dotnet-contribute-code-analysis#contribute-docs-for-caxxxx-rules
+- https://github.com/dotnet/roslyn-analyzers/blob/master/GuidelinesForNewRules.md
+
+If your Pull Request is doing one of the following:
 
 - Adding a new diagnostic analyzer or a code fix
 - Adding or updating resource strings used by analyzers and code fixes
```

Comment:
Are code fix strings included in any of the generated files? Assmuming not, it feels like it could be dropped?

Reply:
I am not sure, I am fine either way (keeping or removing).

Reply:
Either way is fine